### PR TITLE
Delta-1641 Platform > Freedom Components > use FTable from freedom-components

### PR DESCRIPTION
**Pull request by paulyves**
## Ticket references
- [Delta-1641](https://freedom.myjetbrains.com/youtrack/issue/Delta-1641)

## Summary of changes
-  use FTable from freedom-components on existing FTable demo pages

## Checklist

- [ ] bugfix
- [x] new feature
- [ ] breaking change
- [ ] added unit tests
- [ ] changes introduced has been discussed and approved
- [ ] requires manual testing
- [ ] documentation has been updated

## Testing
- Use this configuration for package.json
```
curl https://gist.githubusercontent.com/paulyves/1882b676f65ebf57b2c54772f31ca964/raw | git apply
```
- remove `package-lock.json` and run `npm install`

Check the ff pages:
- [/table-with-tabs](https://localhost:8080/table-with-tabs)
- [/table-with-tabs-2](https://localhost:8080/table-with-tabs-2)
- [/table-without-tabs](https://localhost:8080/table-without-tabs)

PR reference:
- https://gitlab.com/anyTV/incubator/freedom-components/-/merge_requests/57

image

### CODE REVIEW

**Review by pandafox — DISMISSED** [review]

File: src-pwa/pwa-flag.d.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// THIS FEATURE-FLAG FILE IS AUTOGENERATED,
+//  REMOVAL OR CHANGES WILL CAUSE RELATED TYPES TO STOP WORKING
+import "quasar/dist/types/feature-flag";
+
+declare module "quasar/dist/types/feature-flag" {
+  interface QuasarFeatureFlags {
+    pwa: true;
+  }
+}
```
Comment (vcanonjr-freedom) [inline src-pwa/pwa-flag.d.ts:9]: We should not commit this file and `store-flag.d.ts`

Include these in `.gitignore`

/src-pwa/pwa-flag.d.ts
/src/store/store-flag.d.ts

**Review by jonelrempis — DISMISSED** [review]

File: src-pwa/pwa-flag.d.ts (hunk 1 above)
Comment (jonelrempis) [reply src-pwa/pwa-flag.d.ts:9]: @paulyves Remove these files first then add them in a commit.

**Review by vcanonjr-freedom — APPROVED** [review]

**Review by jonelrempis — APPROVED** [review]

**Review by johnurbano12 — APPROVED** [review]
[Test Result Video](https://youtu.be/SN8UNR1Mfts)

**Comment by vcanonjr-freedom** [comment]
Video Reference: https://youtu.be/st6m52-YFeA

**Pull request merged**